src/main/java/org/isf/utils/db/DbSingleJpaConn.java

```diff
@@ -47,7 +47,7 @@ public static Connection getConnection() throws OHException {
 			try {
 				connection = createConnection();
 			} catch (Exception e){
-				String message = MessageBundle.getMessage("angal.utils.dbserverconnectionfailure");
+				String message = MessageBundle.getMessage("angal.utils.databaseserverstoppedornetworkfailure.msg");
```

Comment:
this one looks similar to the one in "angal.sql..."

Reply:
Changed the key to start with "angal.sql."

Reply:
Thanks!